Dockerfile

```diff
@@ -113,5 +113,10 @@ touch /usr/lib/.bootc-dev-stamp
 # And test our own linting
 ## Workaround for https://github.com/bootc-dev/bootc/issues/1546
 rm -rf /root/buildinfo
-bootc container lint --fatal-warnings
+
+dnf -y install 'dnf5-command(copr)'
```

Comment:
This one needs detection for dnf4 vs dnf5

Reply:
with the ostree version check I think we can avoid manually installing the latest ostree. The tests should pass with any ostree version.